util/kubeconfig/kubeconfig_test.go

```diff
@@ -41,7 +41,7 @@ func Test_BasicAuthString(t *testing.T) {
 
 		restConfig, err := clientcmd.RESTConfigFromKubeConfig([]byte(config))
 		assert.NoError(t, err)
-		authString, err := GetAuthString(restConfig)
+		authString, err := GetAuthString(restConfig, "")
```

Comment:
Is there a way to test this with a kubeconfig filepath?

Reply:
kubeconfig path is kubeAPI functionality. Here just passing the text.